package.json

```diff
@@ -12,6 +12,7 @@
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
+    "uuid": "^8.3.2",
```

Comment:
This package is not needed in order to add unique key for list. This is used e.g. for cookies. Please remove it from package.json.